frontend/src/App.css

```diff
@@ -188,4 +188,94 @@
   font-family: Comic Sans MS;
   margin: 1vw;
   font-size: 50pt;
+}
+/* Swiping page */
```

Comment:
Would be good to separate the 'Swiping page' styles into its own css file since it looks quite hefty

Reply:
I agree

Reply:
I put it under app.css because it said unique there and also had the start page styling and results styling so I was unsure of which place to put it

Reply:
Oh yeah I did that too originally but then I noticed that the other pages had its own css file in their respective folders

Reply:
I think so far our code convention has been to keep the styles separate per page, and that only common css (ie. used across multiple pages) should go in App.css. The actual swipe cards are a common component and could have their css in App.css, but the page styling (ie. the buttons for yes/no) should be in page-specific css

Reply:
I see. Ok ill make those changes